jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -121,6 +122,17 @@ function configure_openshift_enhancement() {
     JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.template.cache.ttl=${kscTemplateCacheTTL}"
 }
 
+function configure_workbench_profile() {
+    # Business Central is unified for RHDM and RHPAM; For rhpam-decisioncentral needs to be set org.kie.workbench.profile
+    # to FORCE_PLANNER_AND_RULES and for rhpam-businesscentral and rhpam-businesscentral-monitoring needst to be set to 
+    # FORCE_FULL
+    if [[ $JBOSS_PRODUCT =~ rhdm\-decisioncentral? ]]; then
```

Comment:
The regular expression on this line is incorrect, as the question mark at the end means that the final character (the lowercase letter L) is optional. (Zero or one occurrences.) Instead, no regular expression is needed here, and the check can just be of full string equality:
if [ "$JBOSS_PRODUCT" = "rhdm-decisioncentral" ]; then

The reason that there is a question mark in the business central case is that the entire grouping of (\\-monitoring) can happen zero or one times, so that both "rhpam-businesscentral" and "rhpam-businesscentral-monitoring" would match. The "zero or one occurrence" affects everything in the parenthesis, whereas without the parenthesis, it would only apply to the last character.